docs/user-guide/helm.md

```diff
@@ -50,6 +50,46 @@ source:
     - values-production.yaml
 ```
 
+Argo CD supports Glob expressions for valueFiles (especially useful when using umbrella charts with a lot of dependencies) :
```

Comment:
Could you add a link to the [doublestar language docs](https://github.com/bmatcuk/doublestar?tab=readme-ov-file#patterns) so folks don't assume we're using go's filepath glob?